doc/setup/quickstart_cmake/index.md

```diff
@@ -81,14 +81,17 @@ tutorial
 ├── source
 ├── build
 └── seqan3
-    ├── CMakeLists.txt
-    ├── LICENSE
-    ├── README.md
     ├── build_system
     ├── doc
     ├── include
     ├── submodules
-    └── test
+    ├── test
+    ├── CHANGELOG.md
+    ├── CMakeLists.txt
+    ├── CODE_OF_CONDUCT.md
+    ├── CONTRIBUTING.md
+    ├── LICENSE.md
+    └── README.md
```

Comment:
The ordering doesn't look like what `tree` or `ls` would report?

```cpp
$ tree -L 1 seqan3
seqan3
├── CHANGELOG.md
├── CMakeLists.txt
├── CODE_OF_CONDUCT.md
├── CONTRIBUTING.md
├── LICENSE.md
├── README.md
├── build_system
├── doc
├── include
├── submodules
└── test
```

```bash
$ ls -l
total 44
<metainfo> CHANGELOG.md
<metainfo> CMakeLists.txt
<metainfo> CODE_OF_CONDUCT.md
<metainfo> CONTRIBUTING.md
<metainfo> LICENSE.md
<metainfo> README.md
<metainfo> build_system
<metainfo> doc
<metainfo> include
<metainfo> submodules
<metainfo> test
```

Reply:
I would add the command `tree -L 1 seqan3` just before showing the directory structure itself. 
So the user can call the same command and directly see if everything is up to date :)
(And then adjusting the output like @eseiler suggested)

Reply:
I included the ``` tree ``` command. At this point of the tutorial, the user is in the ``` tutorial ``` directory, and I think that double checking if the ``` source ``` and ``` build ``` directories were successfully created too, is a good idea. Therefore, I used ``` tree -L 2 ```. Also, when running this command, I get another tree structure than suggested above, I've used the one I got.

Reply:
Either I am blind or I do not see a `build` directory in your current update :eyes:  Cna you double check?

Reply:
It's in there :) line 81

Reply:
got it :woman_facepalming: